### PR TITLE
RELEASE-610: PLF 5.1.0-M01

### PR DESCRIPTION
**Pull request by mhbensghaier**

### CODE REVIEW

**Comment by mhbensghaier** [comment]
json modified with platform-private-trial-distributions

File: exo-platform/RELEASE-610.json [hunk 1]
```diff
@@ -0,0 +1,860 @@
...
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.platform.distributions.private.trial.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 13
+    },
+    "release":{
+      "branch":"stable/5.0.x",
+      "version":"5.1.0-M01",
+      "current_snapshot_version":"5.0.x-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: you have to use 5.1.x-SNAPSHOT, for the next line too

File: exo-platform/RELEASE-610.json [hunk 2]
```diff
@@ -0,0 +1,860 @@
...
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.platform.distributions.private.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 12
+    },
+    "release":{
+      "branch":"stable/5.0.x",
+      "version":"5.1.0-M01",
+      "current_snapshot_version":"5.0.x-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: Same as previous and for the next ones too

File: exo-platform/RELEASE-610.json [hunk 3]
```diff
@@ -0,0 +1,860 @@
...
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.answers.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
+      "version":"2.0.0",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 2.1.0-M01

File: exo-platform/RELEASE-610.json [hunk 4]
```diff
@@ -0,0 +1,860 @@
...
+      "current_snapshot_version":"2.0.x-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 2.1.x-SNAPSHOT

File: exo-platform/RELEASE-610.json [hunk 5]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"platform-private-trial-distributions",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.platform.distributions.private.trial.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 13
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: the branch to use is now develop, it's the same for each modules

File: exo-platform/RELEASE-610.json [hunk 6]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"maven-depmgt-pom",
+    "git_organization":"exoplatform",
+    "labels":"pom",
+    "maven_property_version":"org.exoplatform.depmgt.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/13.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 7]
```diff
@@ -0,0 +1,860 @@
...
+    "git_organization":"exoplatform",
+    "labels":"cf",
+    "maven_property_version":"org.exoplatform.cf-parent.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/16.x",
+      "version":"16",
```
[the remark this answers is not in the document]
Comment (mhbensghaier) [reply exo-platform/RELEASE-610.json]: All changes required are done

File: exo-platform/RELEASE-610.json [hunk 8]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.answers.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: should be develop

File: exo-platform/RELEASE-610.json [hunk 9]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.cas.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: should be develop

File: exo-platform/RELEASE-610.json [hunk 10]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.chat.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: should be develop

File: exo-platform/RELEASE-610.json [hunk 11]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.web-conferencing.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.1.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: should be develop

File: exo-platform/RELEASE-610.json [hunk 12]
```diff
@@ -0,0 +1,860 @@
...
+      "version":"1.1.0",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.2.0-M01

File: exo-platform/RELEASE-610.json [hunk 13]
```diff
@@ -0,0 +1,860 @@
...
+      "current_snapshot_version":"1.1.x-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.2.x-SNAPSHOT, the same for the next line

File: exo-platform/RELEASE-610.json [hunk 14]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.cmis.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 15]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.es.embedded.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 16]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.openam.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 17]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.remote-edit.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 18]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.saml.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 19]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.spnego.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 20]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.tasks.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 21]
```diff
@@ -0,0 +1,860 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.web-pack.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/2.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 22]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"addons-manager",
+    "git_organization":"exoplatform",
+    "labels":"exo",
+    "maven_property_version":"org.exoplatform.platform.addons-manager.version",
+    "container": {
+      "image": "exoplatform/release:jdk7-maven32",
+      "step": 1
+    },
+    "release":{
+      "branch": "stable/1.2.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 23]
```diff
@@ -0,0 +1,860 @@
...
+      "version": "1.2.0",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.3.0

File: exo-platform/RELEASE-610.json [hunk 24]
```diff
@@ -0,0 +1,860 @@
...
+      "current_snapshot_version":"1.2.x-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.3.x-SNAPSHOT, same for the next line

File: exo-platform/RELEASE-610.json [hunk 25]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"platform",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.platform.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 10
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 26]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"integration",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.integ.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 9
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 27]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"calendar",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.calendar.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 8
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 28]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"forum",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.forum.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 7
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 29]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"wiki",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.wiki.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 6
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 30]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"social",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.social.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 5
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 31]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"ecms",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.ecms.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 4
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 32]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"commons",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.commons.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 3
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 33]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"platform-ui",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.platform-ui.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 2
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 34]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"docs-style",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.doc.doc-style.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 35]
```diff
@@ -0,0 +1,860 @@
...
+    "labels":"pom",
+    "maven_property_version":"org.exoplatform.depmgt.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"develop",
+      "version":"14-M01",
+      "current_snapshot_version":"13-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 14-SNAPSHOT, idem for the next line

File: exo-platform/RELEASE-610.json [hunk 36]
```diff
@@ -0,0 +1,860 @@
...
+    "git_organization":"exoplatform",
+    "labels":"pom",
+    "maven_property_version":"org.exoplatform.parent.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"develop",
+      "version":"17",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 18-M01

File: exo-platform/RELEASE-610.json [hunk 37]
```diff
@@ -0,0 +1,860 @@
...
+      "current_snapshot_version":"17-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 18-SNAPSHOT, the same for the next line

File: exo-platform/RELEASE-610.json [hunk 38]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"juzu",
+    "git_organization":"juzu",
+    "labels":"juzu",
+    "maven_property_version":"org.juzu.version",
+    "container": {
+      "image": "exoplatform/release:jdk7-maven32",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.2.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 39]
```diff
@@ -0,0 +1,860 @@
...
+      "version":"1.2.0",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.2.0-M01
This version must be release before plf

File: exo-platform/RELEASE-610.json [hunk 40]
```diff
@@ -0,0 +1,860 @@
...
+      "current_snapshot_version":"1.2.x-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.3.x-SNAPSHOT

File: exo-platform/RELEASE-610.json [hunk 41]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"gatein-dep",
+    "git_organization":"exoplatform",
+    "labels":"pom",
+    "maven_property_version":"org.exoplatform.gatein.dep.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.5.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

File: exo-platform/RELEASE-610.json [hunk 42]
```diff
@@ -0,0 +1,860 @@
...
+      "version":"1.5.0",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.6.0-M01

File: exo-platform/RELEASE-610.json [hunk 43]
```diff
@@ -0,0 +1,860 @@
...
+      "current_snapshot_version":"1.5.x-SNAPSHOT",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: 1.6.x-SNAPSHOT, idem for the next line

File: exo-platform/RELEASE-610.json [hunk 44]
```diff
@@ -0,0 +1,860 @@
...
+    "name":"gatein-portal",
+    "git_organization":"exoplatform",
+    "labels":"gatein",
+    "maven_property_version":"org.exoplatform.gatein.portal.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/5.0.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-610.json]: develop

Comment (mhbensghaier) [reply exo-platform/RELEASE-610.json]: required changes done

**Comment by vsellier** [comment]
I think I have fixed the last configuration issues. @thomasdelhomenie  / @mhbensghaier  could you double check please ?

**Pull request closed without merging**